src/http3wtstreamvisitor.h

```diff
@@ -93,8 +93,9 @@ namespace quic
         void tryRead()
         {
             pause_reading_ = false;
-            if (stream_ && stream_->ReadableBytes() > 0)
```

Comment:
May be add instead of `|| can_read_pending_` the check for a fin `|| (sequencer_->IsClosed() && !fin_read_)`.